CMakeLists.txt

```diff
@@ -107,6 +105,22 @@ set(AVIF_PLATFORM_INCLUDES)
 set(AVIF_PLATFORM_SYSTEM_INCLUDES)
 set(AVIF_PLATFORM_LIBRARIES)
 
+# A macro that wraps find_package, preferring static libraries if this is a static build
+# Should only be used for libraries linked against avif, not for those only used
+# by avif_apps or tests
+macro(find_package_libavif)
```

Comment:
Similarly, there is not official way of doing this in CMake, looking for extensions is the current right way of doing it.